### DISCUSSION TITLE
RTL - Seems like RTL option no longer works

### DISCUSSION THREAD

**Discussion by omerman** [katspaugh/wavesurfer.js#3729] [Q&A]
## Bug description
I see you have this example, https://wavesurfer.xyz/example/rtl/
But this example seems like it is not comopatible with V7 of wavesurfer.

Please help?

* P.S it's also not present in the TS declarations

## Environment
 - Browser: Chrome

**Comment by omerman** [comment]
@samanmohamadi What are the chances you re-implement this again on V7 🥹

**Comment by entonbiba** [comment]
#3579 

When I find some time I'll do a pull request for version 7.
All it was doing was the same css as in the reference issue above so faster to simply use the css method if needed for now.

```
if (this.params.rtl === true) {
            if (this.params.vertical === true) {
                util.style(this.container, { transform: 'rotateX(180deg)' });
            } else {
                util.style(this.container, { transform: 'rotateY(180deg)' });
            }
        }
```

**Reply by Binah-CS** [reply]
Will this work for markers, time stamps, etc?

**Reply by omerp-explorium** [reply]
Thank you so much! I will test it and report back 🤩

**Reply by entonbiba** [reply]
@Binah-CS yes, there may be some extra css to add to customize the sizes margins if needed but works pretty well. Let me know how it comes along I'll see what I can do to add it as a feature with some options. @omerp-explorium
